### PR TITLE
set up progress database

### PR DESCRIPTION
**Pull request by charlesinto**
## Questions:

### CODE REVIEW

File: server/Helper.js [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+class Helper {
+    constructor(){
+        this.executeQuery = this.executeQuery.bind(this);
+        this.connectToDb = this.connectToDb.bind(this);
+        // this.assignToken = this.assignToken.bind(this);
+    }
+    // trimWhiteSpace(obj){
+    //     if(typeof obj !== "undefined" && obj !== '' && typeof obj === 'object' && typeof obj.length === "undefined"){
+           
+    //         Object.keys(obj).forEach(function(key){
```
Comment (rovilay) [inline server/Helper.js:20]: Why do you have commented out code??

File: server/model/resources.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        "type":"red-flag",
+        "location":"",
+        "status":"draft",
+        "Images": [],
+        "Videos":[],
+        "comment":"london bridge is falling",
+    },
+    {
+        "id": 13,
+        "createdOn":"23rd Dec, 2018",
```
Comment (rovilay) [inline server/model/resources.js:15]: Good job.

Comment (charlesinto) [reply server/model/resources.js:15]: Thanks

File: server/Helper.js (hunk 1 above)
Comment (charlesinto) [reply server/Helper.js:20]: The functionality is not been used yet, so I don't want it to affect my code coverage

**Pull request merged**